device-types/Cisco/C9500X-28C8D.yaml

```diff
@@ -0,0 +1,247 @@
+---
+manufacturer: Cisco
+model: Catalyst 9500X-28C8D
+slug: cisco-c9500x-28c8d
+part_number: C9500X-28C8D
+u_height: 1
+is_full_depth: true
+comments: '[Cisco Catalyst 9500 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-9500-series-switches/nb-06-cat9500-ser-data-sheet-cte-en.html)'
+front_image: true
+rear_image: true
+console-ports:
+  - name: con0
+    type: rj-45
+  - name: usb
+    type: usb-c
+power-ports:
+  - name: PS-1
+    type: iec-60320-c14
+    maximum_draw: 1500
+  - name: PS-2
+    type: iec-60320-c14
+    maximum_draw: 1500
```

Comment:
These appear to be modular PSU's, is this the case? If so, these should be modelled under `module-bays` and the associated PSU's should be added to the module-types folder if they do not already exist